doc/man/man3/seccomp_rule_add.3

```diff
@@ -147,7 +147,11 @@ value directly using the standard
 values, in order to ensure proper operation across multiple architectures it
 is highly recommended to use the
 .BR SCMP_SYS ()
-macro instead.  See the EXAMPLES section below.
+macro instead.  See the EXAMPLES section below.  If using
+.B __NR_syscall
+values in a context containing non-native architectures, the syscall
+corresponding to the value in the native architecture will be used in all
+non-native architectures.
```

Comment:
I not sure that reads quite right, how about the following?

> ... it is highly recommended to use the SCMP_SYS() macro instead, see the EXAMPLES section below.  It is also important to remember that regardless of the architectures present in the filter, the syscall numbers passed to the libseccomp rule add functions should always be in the context of the native architecture.

Reply:
@Xyene what did you think of the substitute text above?

Reply:
Sorry about the delay, I got DOS'd by real-life stuff -- I've committed a slightly reworded variant of your suggested text, emphasizing that libseccomp is performing some "interpreting" of the given syscall value.